qa/suites/rados/monthrash/ceph.yaml

```diff
@@ -7,3 +7,6 @@ overrides:
 tasks:
 - install:
 - ceph:
+# make sure mgr is up by flushing all pg stats
+- ceph_manager.flush_all_pg_stats:
```

Comment:
How comes this is needed here?  wait-for-healthy is true (I think?) so it would have happened during the ceph task's init